src/recorder.h

```diff
@@ -75,6 +79,12 @@ class Recorder {
   /// returns the unique id associated with this cyclus simulation.
   boost::uuids::uuid sim_id();
 
+  /// returns whether or not the unique simulation id will be injected.
+  bool inject_sim_id() { return inject_sim_id_; };
+
+  /// sets whether or not the unique simulation id will be injected.
+  void set_inject_sim_id(bool x) { inject_sim_id_ = x; };
```

Comment:
I was matching the surrounding code. But yes, I can move this to an overload. And I do hate this pattern :).